pkg/config/default_windows.go

```diff
@@ -20,3 +22,13 @@ func isCgroup2UnifiedMode() (isUnified bool, isUnifiedErr error) {
 func getDefaultProcessLimits() []string {
 	return []string{}
 }
+
+// getDefaultTmpDir for windows
+func getDefaultTmpDir() string {
+	// first check the Temp env var
+	// https://answers.microsoft.com/en-us/windows/forum/all/where-is-the-temporary-folder/44a039a5-45ba-48dd-84db-fd700e54fd56
+	if val, ok := os.LookupEnv("Temp"); ok {
+		return val
+	}
+	return os.Getenv("APPDATA") + "\\Local\\Temp"
```

Comment:
What if `APPDATA` isn't set?  I have no clue about Windows :see_no_evil:

Reply:
AFAIK it is always set, but if not you will likely have a very bad time I assume
We already do this for the containers.conf path.

Reply:
Just to clarify: LOCALAPPDATA, APPDATA go back to Windows XP and TEMP will be defined on all versions. if they aren't someone has altered the env context and we should treat as an error/warning IMO

Reply:
I think you could go either way. If you do use it I would use LOCALAPPDATA over APPDATA since technically it could be moved to somewhere else on some corporate installs, although less common.